### PR TITLE
libpq: set cmake_find_package name

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **libpq/11.5**

See https://github.com/conan-io/conan-center-index/pull/887#discussion_r380733277

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libpq/all/test_package/test_package.c [hunk 1]
```diff
@@ -13,4 +14,4 @@ int main() {
     
     PQfinish(conn);
     return EXIT_SUCCESS;
-}
\ No newline at end of file
+}
```
Comment (SSE4) [inline recipes/libpq/all/test_package/test_package.c]: for @uilianries 
probably, in future we may add hook check for `no new line at the end of file`.
this sometimes emits compiler warnings, and may fail for certain users (e.g. whose compile with **-Werror**).

Comment (Croydon) [reply recipes/libpq/all/test_package/test_package.c]: Yes, please!

Comment (uilianries) [reply recipes/libpq/all/test_package/test_package.c]: Maybe pylint is able to do it, so we could active that hook for conan-center.

Comment (uilianries) [reply recipes/libpq/all/test_package/test_package.c]: Now, I see, do you mean for C and C++ files. Yes, I agree.

Comment (madebr) [reply recipes/libpq/all/test_package/test_package.c]: I would include all common text files. *.py, *.c, *.cpp, *.cc, *.cxx, *.yml, ..
Linters probably have some option for this.
If not, I'ld limit to these file type.

Comment (madebr) [reply recipes/libpq/all/test_package/test_package.c]: I've opened conan-io/hooks#161

**Comment by madebr** [comment]
@danimtb Please give final review and merge :relaxed:

**Pull request merged**